mne/io/egi/tests/test_egi.py

```diff
@@ -288,4 +291,40 @@ def test_io_egi_crop_no_preload():
     assert_allclose(raw._data, raw_preload._data)
 
 
+@requires_version('mffpy', '0.5')
+@requires_testing_data
+@pytest.mark.parametrize('idx, cond, signals, bads', [
+    (0, 'Category 1', egi_txt_evoked_cat1_fname,
+     ['E8', 'E11', 'E17', 'E28', 'ECG']),
+    (1, 'Category 2', egi_txt_evoked_cat2_fname,
+     ['E257', 'EMG'])
+])
+def test_io_egi_evokeds_mff(idx, cond, signals, bads):
+    """Test reading evoked MFF file."""
+    # Test reading all conditions from evokeds
+    evokeds = read_evokeds_mff(egi_mff_evoked_fname)
+    assert len(evokeds) == 2
+    # Test reading evoked data from single condition
+    evoked_cond = read_evokeds_mff(egi_mff_evoked_fname, condition=cond)
+    evoked_idx = read_evokeds_mff(egi_mff_evoked_fname, condition=idx)
+    assert evoked_cond.comment == cond
+    assert evoked_idx.comment == cond
+    # Check signal data
+    data = np.loadtxt(signals, ndmin=2).transpose() * 1e-6  # convert to volts
+    assert_allclose(evoked_cond.data, data, atol=1e-6)
+    assert_allclose(evoked_idx.data, data, atol=1e-6)
+    # Check info
+    assert object_diff(evoked_cond.info, evoked_idx.info) == ''
+    assert evoked_cond.info['description'] == cond
+    assert evoked_cond.info['bads'] == bads
+    assert len(evoked_cond.info['ch_names']) == 259
+    assert 'ECG' in evoked_cond.info['ch_names']
```

Comment:
You can check for channel type existence using `in inst`, e.g.:
```suggestion
    assert 'ECG' in evoked_cond.info['ch_names']
    assert 'ecg' in evoked_cond
```
This assumes that the channel type is set properly to ECG_CH, so if it's not, we can add this later if we correctly detect the channel as type ECG at some point

Reply:
Nice. Just to be clear, do you suggest having both assert statements?
```
assert 'ECG' in evoked_cond.info['ch_names']
assert 'ecg' in evoked_cond
```
or just the one?
```
assert 'ecg' in evoked_cond
```

Reply:
Both, they check different aspects of the info (names and channel types)